app/src/main/kotlin/com/x8bit/bitwarden/MainViewModel.kt

```diff
@@ -377,6 +377,7 @@ class MainViewModel @Inject constructor(
             }
 
             createCredentialRequest != null -> {
+                //TODO only force verification for passkeys??
```

Comment:
You are correct. Passkeys require user verification as part of the FIDO2 spec. The same is not true for passwords, so we do no need to perform user verification for them.